.env.example

```
@@ -0,0 +1,42 @@
+VAPID_SUBJECT=mailto:you@example.com
+NEXT_PUBLIC_VAPID_PUBLIC_KEY=
+VAPID_PRIVATE_KEY=
+
+
+# --- Optional: server-side media cache (seconds) ---
+# Directory used for image byte cache
+IMAGE_CACHE_DIR=/tmp/helprr-image-cache
+# Freshness window for cached images 7 days
+IMAGE_CACHE_TTL_SECONDS=604800
+# Stale-if-error window for cached images # 30 days
+IMAGE_CACHE_STALE_SECONDS=2592000 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)